util/cpp_compat.h

```diff
@@ -0,0 +1,56 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides advanced features not available in C++17
+//
+*******************************************************************************/
+
+#include <map>
+#include <unordered_set>
+
+#if __cplusplus >= 202002L
+#define COMPAT_CPP20 [[deprecated]]
```

Comment:
Perhaps this macro should have a more explicit name, bringing also less risk of conflict?

Reply:
The macro name was replaced with `SYCL_CTS_COMPAT_CPP20` in this commit: 5817d076d8a44b0999706eeb0401abdbf400fa7c